### ISSUE TITLE
Missing C# examples for PropertyTweener class

### ISSUE THREAD

**Issue by hawkerm** [godotengine/godot-docs#9924]
**Your Godot version:** 4.3

**Issue description:**
Was looking for more guidance on how to use `SetCustomInterpolator` in C# as a direct lambda expression can't be used directly inline (`.SetCustomInterpolator(t => Math.Pow(t, 0.75f))`).

Took a bit to get to `.SetCustomInterpolator(Callable.From<float, float>(t => Mathf.Pow(t, 0.75f)))`

Would be nice for the page to have the boxes which switch between gdscript and C# like other examples. Thanks!

**URL to the documentation page (if already existing):** https://docs.godotengine.org/en/stable/classes/class_propertytweener.html#class-propertytweener-method-set-custom-interpolator

**Comment by Bossdell113** [comment]
I saw that this was labeled as a good first issue and I was wondering if I could claim this issue for a school project I am doing.  If I am understanding this correctly the SetCustomInterpolator does not have the C# version in there unlike other methods that have both C# and gdscript.  If this is something I can take my time on then I can totally start working on this, I'd need to do some more research on C# and gdscript though so I won't be able to get this done super quickly, but I think I can manage if given maybe 4 or 5 weeks give or take.  Again, it depends on how much time I'd have for this.

**Comment by raulsntos** [comment]
@Bossdell113 Yes, you can open a pull request to fix this issue, contributions are always welcome :slightly_smiling_face:. As a new contributor, make sure to read [CONTRIBUTING.md](https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md) and the [contributing documentation](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html) if you haven't already.

The issue is C# examples are missing from the documentation in [PropertyTweener.xml](https://github.com/godotengine/godot/blob/master/doc/classes/PropertyTweener.xml). To add C# examples use the `[codeblocks]` syntax, here's an example from [`@GlobalScope.hash`](https://github.com/godotengine/godot/blob/0ed3b47efcb2abe4efff2f58808c944f945bc790/doc/classes/%40GlobalScope.xml#L476-L490):

```xml
<method name="hash">
	<return type="int" />
	<param index="0" name="variable" type="Variant" />
	<description>
		Returns the integer hash of the passed [param variable].
		[codeblocks]
		[gdscript]
		print(hash("a")) # Prints 177670
		[/gdscript]
		[csharp]
		GD.Print(GD.Hash("a")); // Prints 177670
		[/csharp]
		[/codeblocks]
	</description>
</method>
```

Feel free to reach out in the [development chat](https://chat.godotengine.org) if you need help.

**Comment by Bossdell113** [comment]
Thanks for the info.  I'll use the development chat if I have any questions about the issue.
